Mathlib/Algebra/Module/Submodule/Order.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2021 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+
+import Mathlib.GroupTheory.Subgroup.Order
+import Mathlib.GroupTheory.Submonoid.Order
```

Comment:
I deliberately added these imports even though they're possibly redundant, to reduce confusion about mathlib putting orders on stronger structures without weaker structures.

I think we should tell `shake` to ignore these if it complains.